src/common/base64.test.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * @jest-environment jsdom
+ */
+import * as base64 from './base64'
+
+test('can decode an encoded string', () => {
+  const testString = 'ab/c-d=='
+  const encoded = base64.urlEncode(testString)
+  expect(base64.urlDecode(encoded)).toEqual(testString)
```

Comment:
Does it automatically import `test()` and `expect()`?

Reply:
it does, which not everyone is a fan of (this is one of the things `ava` explicitly does differently)